x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/NodeLoad.java

```diff
@@ -261,15 +275,15 @@ public long getFreeMemory() {
         }
 
         public int remainingJobs() {
-            return Math.max(maxJobs - numAssignedJobs, 0);
+            return Math.max(maxJobs - getNumAssignedJobs(), 0);
```

Comment:
I thought we were going to remove pytorch from the job count on the node because its using different executors.

Reply:
Yes, but in the PR where we wire in the planner.